jx_module/main.tf

```diff
@@ -0,0 +1,191 @@
+// ----------------------------------------------------------------------------
+// Enforce Terraform version
+//
+// ----------------------------------------------------------------------------
+terraform {
```

Comment:
Can we try to use terraform 0.13 ?

Reply:
Can we tackle this in a future PR? I consider this a very stable PR the amount of time its been open and don't want to jeopardise that by bumping terraform right just now :)